test/dotnet.Tests/dotnet.Tests.csproj

```diff
@@ -30,7 +30,7 @@
     <PropertyGroup>
       <ExpectedVersionFileInIntermediateFolder>$(IntermediateOutputPath)ExpectedSdkVersion.txt</ExpectedVersionFileInIntermediateFolder>
       
-      <VersionFileContent>$(SdkVersion)</VersionFileContent>
+      <VersionFileContent>$(DotNetCoreSdkLKGVersion)</VersionFileContent>
```

Comment:
I'm having a really hard time understanding what $(SdkVersion) is supposed to represent and where it should be used. It should probably be renamed because it's hard to see that a line like this would be wrong as it was before.

Reply:
@nguerrera 
In the CLI builds $(SdkVersion) is used for the CLI version.